models/articles.models.js

```diff
@@ -55,3 +55,16 @@ exports.addNewComment = (articleID, { username, body }) => {
     .then(({ rows }) => rows[0]);
 };
 
+exports.updateArticle = (articleID, { inc_votes: votes }) => {
+  const keyToUpdate = 'votes';
+  const newValue = votes;
+
+  const updateArticleQuery = `UPDATE articles
+  set ${keyToUpdate} = ${keyToUpdate} + $1
```

Comment:
This is technically fine and isn't opening us up to sql injection but what is the benefit from saving a string of 'votes' to a variable and then using that variable here? Same for newValue = votes? It's fine to utilise variables for dynamic processing but not at the expense of readability which should always be the priority. I know you've done that a few times throughout and can see why, but always remember the priority is readability and that if later there is a benefit to extraction and abstraction then we can do it but not by default.

Something to think about 👍